example/src/App.tsx

```diff
@@ -52,6 +52,64 @@ export default function App({ appName }: { appName: string }) {
         );
         CustomerIO.initialize(cioConfig);
       }
+
+      const logInAppEvent = (name: string, params: InAppMessageEvent) => {
+        console.log(`[InAppEventListener] onEventReceived: ${name}, params: `, params);
+      };
+
+      const onInAppEventReceived = (
+        eventName: string,
+        eventParams: InAppMessageEvent,
+      ) => {
+        logInAppEvent(eventName, eventParams);
+
+        const { deliveryId, messageId, actionValue, actionName } = eventParams;
+        const data: Map<string, any> = new Map();
```

Comment:
This helped me discover `Map` doesn’t work on `main` either. Since this example is copied from old sample app, I fixed it and kept `Map` instead for testing